circularprogressview/src/main/java/com/github/rahatarmanahmed/cpv/CircularProgressView.java

```diff
@@ -256,18 +258,21 @@ public void resetAnimation() {
         if(!isIndeterminate)
         {
             // The cool 360 swoop animation at the start of the animation
-            startAngle = -90f;
-            startAngleRotate = ValueAnimator.ofFloat(-90f, 270f);
-            startAngleRotate.setDuration(5000);
-            startAngleRotate.setInterpolator(new DecelerateInterpolator(2));
-            startAngleRotate.addUpdateListener(new ValueAnimator.AnimatorUpdateListener() {
-                @Override
-                public void onAnimationUpdate(ValueAnimator animation) {
-                    startAngle = (Float) animation.getAnimatedValue();
-                    invalidate();
-                }
-            });
-            startAngleRotate.start();
+            shouldSpin = false;
+            startAngle = typedArray.getFloat(R.styleable.CircularProgressView_cpv_startAngle, -90f);
```

Comment:
This should not check the typedArray, but instead use the startAngle you moved to the top of the file.